turbinia/lib/google_cloud.py

```diff
@@ -35,6 +38,28 @@
 RETRY_MAX = 10
 
 
+def setup_stackdriver_handler(project_id):
+  """Set up Google Cloud Stackdriver Logging
+
+  The Google Cloud Logging library will attach itself as a
```

Comment:
Yep, glad it took only a few lines as well!
* So the closest info I was able to find re stackdriver logging performance is: https://medium.com/google-cloud/python-and-stackdriver-logging-2ade460c90e3. TLDR is that we're implementing it in a way where we're attaching the stackdriver logging client as a handler to Python logging which sends logs asynchronously/via a background process and does not block the main program.
* Since it's attaching itself as a handler and what I mentioned above, I'm thinking it should still continue, but that'll be a bit hard to test. I'm thinking the only situations we'd only run into an issue is where the logging API service is down/a bug in GCP logging code where it isn't able to properly send logs, but that's more on GCP side. I can't think of a fail safe to add on Turbinia in those situations.